src/database/postgres/insert_query_data.md

```diff
@@ -0,0 +1,49 @@
+## Insert and Query data
+
+[![postgres-badge]][postgres] [![cat-database-badge]][cat-database]
+
+The recipe inserts data into the `author` table. Then, displays the data from the `author` table.
```

Comment:
What method inserts data?  What method queries data?  These need to be answered with links in the description.